lib/ansible/modules/cloud/amazon/ec2_group.py

```diff
@@ -149,15 +149,17 @@
           - 443
           - 8080-8099
         cidr_ip: 0.0.0.0/0
+      # rule source list support was added in version 2.3, as well as multiple source types in the same rule.
+      - proto: tcp
+        ports:
+          - 6379
+          - 26379
+        group_name:
+          - example-vpn
+          - example-redis
+        cidr_ip: 172.16.1.0/24
```

Comment:
This is great, I really like this.

Can you make it clearer that all source types are now supported (group_name, group_id and cidr_ip) - it's clear to me from reading the source code, but wasn't when I looked at this example (might be worth having separate examples for different source types?)